documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue

```diff
@@ -1613,21 +1613,21 @@ export default {
       return this.getFolderPath(this.folderPath);
     },
     showVersionPreview(version) {
-      return this.$attachmentService.getAttachmentById(version.originId)
+      return this.$attachmentService.getDocumentDetails(version.originId)
         .then(attachment => {
-          return this.$attachmentService.getAttachmentById(version.frozenId)
+          return this.$attachmentService.getDocumentDetails(version.frozenId)
             .then(file => {
-              file.downloadUrl =`${attachment.downloadUrl}?version=${version.versionNumber}`;
+              file.downloadUrl =`/rest/jcr/repository/collaboration${attachment.path}?version=${version.versionNumber}`;
```

Comment:
Maybe use for the future URLs, the new WebDav URL, but not for now. In fact, this URL has to be deprecated and then deleted.